sql/src/test/java/ddlog/WeaveTest.java

```diff
@@ -12,7 +12,7 @@ public void testWeave() {
         Translator t = new Translator(null);
         String node_info = "create table node_info\n" +
                 "(\n" +
-                "  name varchar(36) not null /*primary key*/,\n" +
+                "  name varchar(36) not null with (primary_key = true),\n" +
```

Comment:
Can you also test for multiple primary keys in the same table (compound key)?

Reply:
See keyTest